flyway-core/src/main/java/org/flywaydb/core/internal/database/cockroachdb/CockroachDBTable.java

```diff
@@ -30,6 +30,7 @@
  * carrying out a migration.
```

Comment:
note: if the locking strategy does change, the comment here should be updated. But I'm not sure if the locking can be changed right now.